plaso/lib/utils.py

```diff
@@ -32,19 +32,21 @@ def IsText(bytes_in, encoding=None):
   # Start with the assumption we are dealing with a text.
   is_ascii = True
 
+  if isinstance(bytes_in, py2to3.UNICODE_TYPE):
+    return is_ascii
```

Comment:
lets rename `is_ascii` to `is_text` seeing an ASCII Unicode string does not make sense to me

Reply:
or define `is_ascii` after this check and return True